extensions/ql-vscode/src/data-extensions-editor/external-api-usage-query.ts

```diff
@@ -78,18 +51,42 @@ export async function runQuery(
     name: "codeql/external-api-usage",
     version: "0.0.0",
     dependencies: {
-      [`codeql/${databaseItem.language}-all`]: "*",
+      [`codeql/${language}-all`]: "*",
     },
   };
 
   const qlpackFile = join(queryDir, "codeql-pack.yml");
-  await writeFile(qlpackFile, dumpYaml(syntheticQueryPack), "utf8");
+  await writeFile(qlpackFile, dump(syntheticQueryPack), "utf8");
+}
+
+export async function runQuery(
+  mode: Mode,
+  {
+    cliServer,
+    queryRunner,
+    databaseItem,
+    queryStorageDir,
+    queryDir,
+    progress,
+    token,
+  }: RunQueryOptions,
+): Promise<CoreCompletedQuery | undefined> {
+  // The below code is temporary to allow for rapid prototyping of the queries. Once the queries are stabilized, we will
```

Comment:
The note about the queries moving to the github/codeql repo makes sense. I'm not 100% sure what "non-pretty" aspect of the code the comment is referring to, and it's a little hard to compare against the code linked in the comment to see what's different.

This comment could maybe be made clearer for new readers, but alternatively it's fine to just leave it if it's too hard. As you say, the code should be deleted soon-ish hopefully.

Reply:
In this case here I only moved the comment (and code) to a new place, it existed before.